externalsecret.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: vault-example
+  namespace: external-secrets
+spec:
+  refreshInterval: "15s"
+  secretStoreRef:
+    name: vault-backend
+    kind: ClusterSecretStore
+  target:
+    name: test-vault-secret
+  data:
+  - secretKey: test_user
+    remoteRef:
+      key: kv/data/test-secret
+      property: test_user
```

Comment:
What exactly is this? A set of manifests for test? If so, let's keep it in the test folder, not in the root